Sources/SQL/Serialization/SQLSerializer+DataQuery.swift

```diff
@@ -31,7 +31,7 @@ extension SQLSerializer {
             }
 
             if columns.isEmpty {
-                columns += ["*"] // ["\(table).*"]
+                columns += query.joins.isEmpty ? ["*"] : ["\(table).*"]
```

Comment:
Why not always `["\(table).*"]`?

Reply:
apparently because db kit needs to be able to select even the join table stuff in case you got some more info on it ... Tanner says there is a bug in Fluent on the actual mapping @MrMage